editor/plugins/game_view_plugin.cpp

```diff
@@ -235,6 +243,13 @@ void GameView::_hide_selection_toggled(bool p_pressed) {
 	debugger->set_selection_visible(!p_pressed);
 }
 
+void GameView::_debug_mute_audio_button_pressed() {
+	debug_mute_audio = !debug_mute_audio;
+	debug_mute_audio_button->set_button_icon(get_editor_theme_icon(debug_mute_audio ? SNAME("AudioMute3") : SNAME("AudioStreamPlayer")));
```

Comment:
Remember to rename the icon file you're using here to `AudioMute` and remove the other two icons.

Reply:
Thank you for catching that! I'll make sure to get that updated.